src/tests/unit/tests/reports/package/reporter-factory.test.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I'm wondering if we should have a `integration` folder for this test...

Reply:
That makes sense. I didn't want to add that burden to this initial PR but I would like to have several scan files with expected results.

Reply:
Discussed with @smoralesd and agreed that at a minimum these tests should be in a file named integration for now. Deferring on creating a new category of tests for now given that these tests are still very quick to run.